server/__init__.py

```diff
@@ -126,15 +126,33 @@
     }
 ]
 # Dict of production models, key: model name, value: tuple (clf, generator_fn, predictor_fn).
+# Changed to just load the defualt model
 production_models = {}
 for model_config in MODEL_CONFIGS:
-    print("Loading: " + model_config["name"])
-    clf = joblib.load(model_config["path"])
+    print("Putting in dictionary: " + model_config["name"])
     gen = model_config["generator"]
     pred = model_config["predictor"]
-    production_models[model_config["name"]] = (clf, gen, pred)
+    if model_config["name"] == DEFAULT_MODEL:
+        print("Loading: " + model_config["name"])
+        clf = joblib.load(model_config["path"])
+        production_models[model_config["name"]] = (clf,gen, pred)
+    else:
+        production_models[model_config["name"]] = (None, gen, pred)
 print("\tDone in {:.1f} secs".format(time.time() - start_time))
 
+#Load the called model function
+
+def load_model(name):
+    print("Loading: " + model_config["name"])
+    for model_config in MODEL_CONFIGS:
```

Comment:
And instead of doing this loop, get the path you saved in the tuple and use it to load the model.

Then `load_model` can run in constant time instead of linear time for the number of models.